rero_ils/modules/loans/api.py

```diff
@@ -159,23 +159,30 @@ def organisation_pid(self):
         if self.get('item_pid'):
             item = Item.get_record_by_pid(self.get('item_pid'))
             return item.organisation_pid
-        return None
 
     @property
     def library_pid(self):
         """Get library PID regarding transaction location PID or location."""
         from ..items.api import Item
 
         location_pid = self.get('transaction_location_pid')
+        if location_pid:
```

Comment:
can replace of these by:
return Location.get_record_by_pid(self.location_pid).library_pid
?

Reply:
done